services/brig/test/integration/API/Provider.hs

```diff
@@ -1561,15 +1653,15 @@ prepareBotUsersTeam brig galley sref = do
     let sid = sref^.serviceRefId
 
     -- Prepare users
-    u1 <- createUser "Ernie" "success@simulator.amazonses.com" brig
-    u2 <- createUser "Bert"  "success@simulator.amazonses.com" brig
-    let uid1 = userId u1
+    (uid1, tid) <- Team.createUserWithTeam brig galley
+    u1 <- selfUser <$> getSelfProfile brig uid1
+    u2 <- Team.createTeamMember brig galley uid1 tid Team.fullPermissions
```

Comment:
It turned out that `addTeamMember` wasn't updating user's team in Brig, which made sense but also was wrong. Now, whenever we want to create a team member in tests, we do it by using `/register` and the user becomes a member of the team right from the start.